frontend/src/metabase/nav/components/ProfileLink/utils.ts

```diff
@@ -0,0 +1,27 @@
+import type { Settings } from "metabase-types/api";
+
+export const getHelpLink = ({
```

Comment:
I think maybe this function could be converted to a React hook and move some of the values we get from hook and pass to this function into this function's body instead to help reduce the number of parameters passed to this function.